docs/products/mysql/concepts/high-availability-mysql.md

```diff
@@ -0,0 +1,112 @@
+In the event of database failover, the **Service URI** of your service
+remains the same; only the IP address will change to point to the new
+primary node.
+:::
+
+## Highly available Business and Premium service plans
+
+When the failed node is a MySQL standby node, the primary node
+keeps running normally and provides a normal service level to the client
+applications. Once the new replacement standby node is ready and
+synchronised with the primary node, it starts replicating the primary
+node in real time as the situation reverts back to normal.
+
+When the failed node is a MySQL primary node, the combined
+information from the Aiven monitoring infrastructure and the standby
+node is used to make a failover decision. The standby node is then
+promoted as the new primary and immediately starts serving clients. A
+new replacement node is automatically scheduled and becomes the new
+standby node. There is no data loss in this scenario.
+
+If all the primary and standby nodes fail at the same time or if all the primary
+nodes fail while standby nodes are being recovered, new
+nodes are automatically scheduled for creation to become the new primary
+and standby. The primary node is restored from the latest available
+backup, which can involve some degree of data loss. Any write
+operations made since the backup of the latest write-ahead logging (WAL) file are lost.
+Typically, this time window is limited to either five minutes of time or one WAL file.
+
+If all the primary nodes fail while a standby is being recovered
+
+:::note
+The amount of time it takes to replace a failed node depends mainly on
+the selected cloud region and the amount of data to be restored.
+However, in the case of partial loss of the cluster, the surviving node
+keeps on serving clients even during the recreation of the other node.
+All of this is automatic and requires no administrator intervention.
+:::
+
+**Premium** plans operate in a similar way as **Business** plans. The
+main difference comes when one of the standby or primary nodes
+fails. Premium plans have an additional, redundant standby node
+available, providing platform availability even in the event of losing
+two nodes. In cases where the primary node fails, Aiven monitoring tool determines which
+of the standby nodes is the furthest along
+in replication (has the least potential for data loss) and does a
+controlled failover to that node.
+
+:::note
+For Aiven for MySQL backups and restoration, Aiven uses
+[MyHoard](https://aiven.io/blog/introducing-myhoard-your-single-solution-to-mysql-backups-and-restoration).
+:::
+
+## Single-node Hobbyist and Startup service plans
+
+Hobbyist and Startup plans provide a single node; when it's lost, Aiven
+immediately starts the automatic process of creating a new replacement
+node. The new node starts up, restores its state from the latest
+available backup, and resumes serving customers.
+
+Since there is just a single node providing the service, the service is
```

Comment:
I think this paragraphs need a rewrite with engineering input. MySQL doesn't use WAL files (even though sometimes WAL is misused interchangeably).
cc: @alexole